src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const RENT = 40;
+  const SMALL_DISCOUNT = 20;
+  const BIG_DISCOUNT = 50;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+
+   if (days >= LONG_TERM) {
+    return (days * RENT) - BIG_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
```

Comment:
DON'T add else after if with return - the code after it won't be executed anyway
Check checklist